docs/source/contributing.rst

```diff
@@ -35,58 +35,62 @@ Installing a Development Version
 --------------------------------
 
 
-.. todo::
+1. Fork both the SCICO repository and the SCICO Data submodule by clicking the Fork button on the `repository page <https://github.com/lanl/scico>`_. This will create a copy of both SCICO and SCICO Data repositories in your Git account.
 
-   At time of public release, this should be updated to a forking tutorial (see
-   `the jax example <https://jax.readthedocs.io/en/latest/contributing.html#contributing-code-using-pull-requests>`_)
 
+2. Make sure that you have Python >= 3.8 installed in order to create a conda virtual environment.
 
-1. Create a conda environment using Python >= 3.8:
+
+3. To create a conda virtual environment, clone your fork from source. To do a clone on SCICO that includes the SCICO Data submodule:
 
 ::
 
-   conda create -n scico python=3.8
+   git clone --recurse-submodules git@github.com:<username>/scico.git
 
 
-2. Activate the conda virtual environment:
+4. Add the SCICO repo as an upstream remote to sync your changes:
 
 ::
 
-   conda activate scico
+   git remote add upstream https://www.github.com/lanl/scico
+
+
+5. After adding the upstream, the recommended way to install SCICO and its dependencies is via `conda <https://docs.conda.io/en/latest/>`_ using the scripts in ``misc/conda``:
+
+  - ``install_conda.sh``: install ``miniconda``
+    (needed if conda is not already installed on your system).
+  - ``conda_env.sh``: install a ``conda`` environment
+    with all SCICO dependencies. For GPU installation, see ``conda_env.sh -h``.
 
-3. Clone the SCICO repository:
+
+5. Activate the created conda virtual environment:
 
 ::
 
-   git clone https://github.com/lanl/scico.git --recurse-submodules
+   conda activate py38
 
 
-4. Navigate to the cloned repository:
+6. Navigate to the root of the cloned repository:
 
 ::
 
     cd scico
 
-5. Install dependencies:
+7. Once dependencies have been installed, install SCICO in editable form:
 
 ::
 
-  pip install -r requirements.txt  # Installs basic requirements
-  pip install -r docs/docs_requirements.txt # Installs documentation requirements
-  pip install -r examples/examples_requirements.txt # Installs example requirements
-  pip install -e .  # Installs SCICO from the current directory in editable mode.
```

Comment:
My want to restore these, again, per discussion with @Michael-T-McCann

Reply:
Yep! Will do.

Reply:
Decision made to keep these and drop the install script recommendation.